### PR TITLE
add examples for bizops

### PR DESCRIPTION
**Pull request by sqs**
@dadlerj @nicksnyder  These are examples from our chat today.

@dadlerj 	Please review and change as needed.

### CODE REVIEW

**Comment by dadlerj** [comment]
A few thoughts on this, will add a PR later today

**Comment by sqs** [comment]
@dadlerj ping on this

File: job-descriptions/growth-biz-ops.md [hunk 1]
```diff
@@ -28,6 +28,15 @@ You will:
 - Be data-driven in helping our go-to-market teams (Developer Advocates and Sales) prioritize opportunities to invest their time.
 - Help build a growth and business team from the ground up.
 
+#### Examples
+
+Our engineering, go-to-market, and product teams will rely on you when making important decisions. Some examples of the kinds of questions you'll answer for the team are:
+
+- What programming languages do our customers use? How does product adoption and satisfaction differ by language?
+- What kinds of search queries are yielding successful results? No results? (You'll need to measure this using anonymized features of search queries, such as a boolean indicating whether the query contains a regular expression or just a literal string, to preserve user privacy.)
```
Comment (ryan-blunden) [inline job-descriptions/growth-biz-ops.md]: This seems like it shouldn't be in parens.

File: job-descriptions/growth-biz-ops.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ You will:
...
+- How are our customers deploying their Sourcegraph instance (AWS, Google Cloud, Azure, DigitalOcean, etc.)?
```
Comment (ryan-blunden) [inline job-descriptions/growth-biz-ops.md]:
```suggestion
- How are customers deploying Sourcegraph (AWS, Google Cloud, Azure, DigitalOcean, etc.)?
```

**Pull request closed without merging**